doc/nrf/protocols/matter/end_product/ecosystems_certification.rst

```diff
@@ -0,0 +1,61 @@
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter stack provided in the |NCS| works with commercial ecosystems that are compatible with official Matter implementation.
+The Matter product is called compatible with Matter implementation once it passed relevant Matter certification.
+
+However some of the ecosystem providers require passing additional certification to verify that Matter product is able to work with their applications and support their features.
+Once the Matter product passes the ecosystem related certification, the appropriate information can be placed on the product's packaging.
+
+.. _ug_matter_google_certification:
+
+Works with Google Home certification
+************************************
+
+The Google Home requires passing the `Works with Google Home`_ certification test cases to ensure the Matter product supports Matter features in the same extent, as the Google Home application.
+The product that passed the certification is allowed to use the Works with Google Home badge.
+
+The Matter samples delivered in the |NCS| did not receive official certificates, as these can be obtained only for the final products, however they were tested against the Works with Google Home certification test cases and they successfully passed all of them.
+The following list contains a full list of Matter samples that were verified to pass Works with Google certification test cases:
+
+* :ref:`Matter door lock <matter_lock_sample>`
+* :ref:`Matter light bulb <matter_light_bulb_sample>`
+* :ref:`Matter thermostat <matter_thermostat_sample>`
+* :ref:`Matter window covering <matter_window_covering_sample>`
+* :ref:`Matter weather station <matter_weather_station_app>`
+
+.. _ug_matter_amazon_certification:
+
+Amazon Frustration-Free Setup certification
+*******************************************
+
+Frustration-Free Setup (FFS) is Amazon's proprietary feature that uses Amazon cloud service to provide onboarding information necessary to commission the Matter device to a Matter fabric.
+The commissioning process is automatically triggered by the Amazon commissioner after detecting accessory's Bluetooth LE advertising messages supporting proprietary format and it does not require any user interaction.
+
+For more information about the FFS and how to enable it in Matter samples, see :ref:`Amazon Frustration-Free Setup configuration<ug_matter_configuring_ffs>` page.
+
+FFS requires passing the additional certification test cases for Matter called `Matter Simple Setup`_.
+There are following two types of certification:
+
+* Default - for standard products or for the creation of a reference design
+* Simplified - for products using a certified reference design
+
+The Matter platform provided in the |NCS| was tested against the FFS certification test cases and passed them in several variants.
+If you create a Matter product based on |NCS|, you can benefit from it and approach the simplified certification path.
```

Comment:
Could we please link to the Amazon certification page? Since it would be nice to understand what "simplified" means - i.e. less testing, no need to send devices to Amazon for lab testing.

Reply:
We link to the Amazon certification page in this sentence `FFS requires passing the additional certification test cases for Matter called `Matter Simple Setup`_.` Do you think it is enough or should we point to some specific part of the page?